include/utils.php

```diff
@@ -5843,3 +5843,22 @@ function getAppString($key)
 
     return $app_strings[$key];
 }
+
+function get_upload_dir() {
+
+    $config = SugarConfig::getInstance();
+
+    if (empty($config)) {
+        return 'upload/';
```

Comment:
the 'upload/' string can be taken into a variable say: $defaultUploadDir = 'upload/'; as the string is repeating a few times